src/serializers/json-api.js

```diff
@@ -0,0 +1,22 @@
+//TODO: Implement 'included' support after relationships
```

Comment:
I would like it to be named same as class => `JSONApiSerializer`, what do you think? :)